src/config_objects.c

```diff
@@ -762,6 +764,19 @@ TbBool parse_objects_object_blocks(char *buf, long len, const char *config_textn
                     CONFWRNLOG("Incorrect value of \"%s\" parameter in [%s] block of %s file.",
                         COMMAND_TEXT(cmd_num), block_buf, config_textname);
                 }
```

Comment:
like github says, this statement may fall through 
you need a break at the end of a case

Reply:
Thanks, fixed.